distribution/std-lib/Standard/src/Examples.enso

```diff
@@ -181,3 +163,96 @@ image = Enso_Image.read here.image_file [Codecs.Read_Alpha_Channel]
+    [row_1, row_2, row_3].to_json
+
+## The headers for the columns in the JSON table `here.simple_table_json`.
+simple_table_json_headers : Vector Text
+simple_Table_json_headers = ["foo", "bar", "baz"]
+
+## Some simple GeoJSON.
+geo_json : Enso_Json.Json
+geo_json = Enso_Json.parse <| '''
+    {
+      "type": "FeatureCollection",
+      "features": [
+        {
+            "type": "Feature",
+            "properties": { "foo": 1, "bar": "value2"},
+            "geometry": { "type": "Point", "coordinates": [ -118.58, 34.19 ] }
+        },
+        {
+            "type": "Feature",
+            "properties": { "foo": 2, "baz": 3},
+            "geometry": { "type": "Point", "coordinates": [ 10.11, 4.88, 19 ] }
+        }
+      ]
+    }
+
+## A small table column containing integers.
+integer_column : Table.Column.Column
+integer_column = Table.Column.from_vector "Integer" [1, 4, 8, 2, 5]
+
+## A small table column containing decimal numbers.
+decimal_column : Table.Column.Column
+decimal_column = Table.Column.from_vector "Decimal" [2.30, -2.1, Nothing, -10.1, 1.0]
+
+## A small table column containing booleans.
+bool_column_1 : Table.Column.Column
+bool_column_1 = Table.Column.from_vector "Bools" [True, True, False, True, False]
+
+## A small table column containing booleans.
+bool_column_2 : Table.Column.Column
+bool_column_2 = Table.Column.from_vector "Bools" [False, True, Nothing, True, True]
+
+## A small table column containing text.
+text_column_1 : Table.Column.Column
+text_column_1 = Table.Column.from_vector "Text" ["Hello,", "my", "name", "is", "Enso"]
+
+## A small table column containing text.
+text_column_2 : Table.Column.Column
+text_column_2 = Table.Column.from_vector "Text" ["He,", "he", "he", "i", "so"]
+
+## A simple aggregate column.
+aggregate_column : Table.Column.Aggregate_Column
+
+## A simple table that contains basic shop inventory data for the food shop.
+inventory_table : Table.Table
+inventory_table = Table.from_csv here.csv . set_index "item_id"
+
+## A simple table that contains basic item popularity data for the food shop.
+popularity_table : Table.Table
+popularity_table =
+    table = Table.from_csv <| Enso_Project.data / "food_shop_popularity.csv"
+    table.set_index "item_id"
+
+## A simple tablethat contains basic transaction data for the food shop.
+transactions_table : Table.Table
+transactions_table =
+    Table.from_csv <| Enso_Project.data / "food_shop_transactions.csv"
+
+## An aggregate table for the relevant examples.
+aggregate_table : Table.Aggregate_Table
+aggregate_table =
+    transactions = Examples.transactions_table
+    item_names = Examples.inventory_table.at "item_name"
+    with_names = transactions.join item_names on="item_id"
+    with_names.group by="item_name"
+
+## An aggregate column for the relevant examples.
+aggregate_column : Table.Column.Aggregate_Column
+aggregate_column = here.aggregate_table.at "transaction_id"
```

Comment:
I noticed that examples becoming more complex. Until we have automatic doctests, it would be nice to have an `Examples_Spec` checking that functions are at least executable